src/index.js

```diff
@@ -33,7 +33,7 @@ import Reviews from './components/pages/Reviews/MentorReviews';
 import Notes from './components/pages/Notes/Notes';
 import NotesForm from './components/pages/Notes/NotesForm';
 import Attendance from './components/pages/Attendance/attendance';
-import MentorAddReview from './components/pages/AddReviews/MentorAddReview';
+// import MentorAddReview from './components/pages/AddReviews/MentorAddReview';
```

Comment:
I know that this must be commented out for the code to work, but I wonder if leaving it uncommented is best for the merge to main since it will be fixed soon? Not sure what the best move is here but understandable why you would leave it commented out.

Reply:
I uncommented these lines since there is a fix for the error these were causing coming soon.